app/src/main/java/org/simlar/https/SimlarSSLSocketFactory.java

```diff
@@ -94,8 +94,7 @@ private static SSLSocketFactory createSSLSocketFactory()
 			context.init(null, tmf.getTrustManagers(), null);
 			return context.getSocketFactory();
 		} catch (final CertificateException | NoSuchAlgorithmException | IOException | KeyStoreException | KeyManagementException e) {
-			Lg.ex(e, "Exception during createSSLSocketFactory");
-			return null;
+			throw new RuntimeException("unable to create SSL socket", e);
```

Comment:
"the idiom can only be used when the construction of Something can be guaranteed to not fail"
https://en.wikipedia.org/wiki/Initialization-on-demand_holder_idiom
Now, it can fail.
Besides, createCipherSuites() and createProtocols() may return `null`, too.